lib/fontcustom/scripts/generate.py

```diff
@@ -77,8 +77,8 @@ def createGlyph( name, source, code ):
             aligned_to_pixel_grid = (width % design_px == 0)
             if (aligned_to_pixel_grid):
                 shift = glyph.left_side_bearing % design_px
-                glyph.left_side_bearing = glyph.left_side_bearing - shift
-                glyph.right_side_bearing = glyph.right_side_bearing + shift
+                glyph.left_side_bearing = int(glyph.left_side_bearing) - int(shift)
```

Comment:
This is the change I had to make to make fontcustom compile. I didn't do any verification on if this is correct, this was just to make the compilation finish, so I don't think I should include this change?